server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -2083,31 +2090,41 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
 
         Map<String, String> configs = _configDao.getConfiguration("AgentManager", params);
 
-        _kvmMtuSize = NumbersUtil.parseInt(_configDao.getValue(KVM_MTU_KEY), 0);
+        final List<ClusterVO> clusterVOs = _clusterDao.listByHypervisorAndCluster(HypervisorType.KVM, Cluster.ClusterType.CloudManaged);
+
+        if ((clusterVOs != null) && !clusterVOs.isEmpty()) {
```

Comment:
CollectionUtils.isEmpty(clusterVOs) would be nice here :slightly_smiling_face:  :+1: